pkg/plugins/generic/generic_plugin.go

```diff
@@ -212,3 +196,31 @@ func needDrainNode(desired sriovnetworkv1.Interfaces, current sriovnetworkv1.Int
 	}
 	return
 }
+
+func needRebootNode(state *sriovnetworkv1.SriovNetworkNodeState, loadVfioDriver *uint) (needReboot bool) {
+	needReboot = false
+	if *loadVfioDriver != loaded {
+		if needVfioDriver(state) {
+			*loadVfioDriver = loading
+			update, err := tryEnableIommuInKernelArgs()
+			if err != nil {
+				glog.Errorf("generic-plugin needRebootNode():fail to enable iommu in kernel args: %v", err)
+			}
+			if update {
+				glog.V(2).Infof("generic-plugin needRebootNode(): need reboot for enabling iommu kernel args")
+			}
+			needReboot = needReboot || update
+		}
+	}
+
+	update, remove, err := utils.WriteSwitchdevConfFile(state)
```

Comment:
Why we need to distinguish  'remove' from 'update' here?

Reply:
We don't, will update.

Reply:
Done